README.md

```diff
@@ -62,34 +55,176 @@ notifications_client.send_sms_notification(
 )
 ```
 
+Find `template_id` by clicking **API info** for the template you want to send.
+
+If a template has placeholders, you need to provide their values in `personalisation`.
+
+<details> 
+    <summary>
+        Response:
+    </summary>
+
+<table>
+  <thead>
+    <tr>
+        <td>Status
+        </td>
+        <td>Body
+        </td>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+        <td>201
+        </td>
+        <td>
+<pre>
+{
+  "data":{
+    "notification": {
+      "id":1
+    },
+    "body": "Dear Bill, your licence is due for renewal on 3 January 2016.",
+    "template_version": 1,
+    "subject": "Licence renewal"
+  }
+}
+</pre> 
+        </td>
+    </tr>
+    <tr>
+        <td>429
+        </td>
+        <td>
+<pre>
+{"result": "error",
+"message": "Exceeded send limits (50) for today"}
+</pre>
+        </td>
+    </tr>
+        <tr>
+        <td>400
+        </td>
+        <td>
+<pre>
+{"result":"error",
+"message":{"template": ["Missing personalisation: {template_placeholder_name}"]} 
+</pre>
+        </td>
+    </tr>
+        </tr>
+        <tr>
+        <td>400
+        </td>
+        <td>
+<pre>
+{"result":"error", 
+"message"={"to": ["Invalid {notification_type} for restricted service")]}
```

Comment:
we've changed these messages - there's now a couple of messages they might see instead (still with 400 response code):

`"Can’t send to this recipient using a team-only API key"`

or if they're in trial mode 

`"Can’t send to this recipient when service is in trial mode – see https://www.notifications.service.gov.uk/trial-mode"`